src/exercises/day-1/morning.md

```diff
@@ -5,3 +5,15 @@ In these exercises, we will explore two parts of Rust:
 * Implicit conversions between types.
 
 * Arrays and `for` loops.
+
+<details>
+
+A few things to consider while solving the exercises:
+
+* Use a local Rust installation, if possible. This way you can get auto-completion in your editor.
```

Comment:
Would it be useful for this to link to https://rustup.rs/?

Reply:
Yes, good idea! I linked back to our installation instructions so that we keep it all in one place.